airbyte-cdk/java/airbyte-cdk/core/build.gradle

```diff
@@ -58,6 +58,9 @@ dependencies {
     api 'org.jooq:jooq:3.16.23'
     api 'org.apache.commons:commons-csv:1.10.0'
 
+    compileOnly 'org.projectlombok:lombok:1.18.30'
```

Comment:
should we define this in deps.toml file and refer it in each individual `build.gradle` file?

Reply:
As it turns out, @stephane-airbyte got rid of the lombok dependency entirely, so I'm going to update this PR accordingly.

Reply:
Yes, eventually. We should clean up the deps.toml file at some point, so that it only includes dependencies like lombok, junit, etc. which we're VERY CONFIDENT they're are not going to change the connector docker images meaningfully when bumped.